### ISSUE TITLE
my-members presenceMap requirement change for 1.1

### ISSUE THREAD

**Issue by SimonWoolf** [ably/ably-java#397]
See https://github.com/ably/docs/pull/446

**Comment by SimonWoolf** [comment]
@amsurana Just want to check in on the status of this? Bit worried that this issue seems to be left open after the 1.1 release -- as noted in the linked docs pr, the server assumes that any client library that sends `v=1.1` implements the local presence map up to spec

**Comment by amsurana** [comment]
@SimonWoolf 
No, This feature is not implemented yet. I will look at it soon.

**Comment by amsurana** [comment]
@SimonWoolf 
I will check this and confirm here.

**Comment by amsurana** [comment]
@SimonWoolf 
I will check this and confirm here.

**Comment by paddybyers** [comment]
The spec has been updated: https://github.com/ably/docs/pull/711.

Currently this library advertises a protocol version of `1.0` - this requirement must be met before updating the protocol to `1.0`.
